provisioning/playbook.yml

```diff
@@ -100,6 +100,11 @@
       set_fact:
         drupal_site_exists: "{{ drupal_site.stat.exists|default(false) }}"
 
+    - name: Update drush_path if drush is not available globally.
+      set_fact:
+        drush_path: "{{ drupal_composer_install_dir }}/vendor/drush/drush/drush"
+      when: "'drush' not in installed_extras"
```

Comment:
Due to [variable precedence](http://docs.ansible.com/ansible/playbooks_variables.html#variable-precedence-where-should-i-put-a-variable) this will override the role default but can still be overridden in `config.yml` if set. I think? I'll test it and make sure a bit later.